Makefile

```diff
@@ -1,3 +1,11 @@
+.PHONY: test integration-tests scale-tests
 
-test:
-	PYTHONWARNINGS=ignore:ResourceWarning python -m unittest discover
+test: integration-tests
+
+integration-tests:
+	PYTHONWARNINGS=ignore:ResourceWarning python -m unittest \
+		discover --start-directory tests/integration --top-level-directory . --verbose
+
+scale-tests:
+	PYTHONWARNINGS=ignore:ResourceWarning python -m unittest \
+		discover --start-directory tests/scale --top-level-directory . --verbose
```

Comment:
Does this mean 1 make will trigger both int and scale tests? Is it configurable via Travis?

Reply:
Not yet.  The mechanism by which we run the scale test is undecided.  I think this requires discussion, as, for example it can leave many workflows stuck in the green box burning dollars.  I'm thinking that perhaps the scale test should never be run unsupervised.  Perhaps we just run it once a week manually with supervision.